Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -131,6 +149,12 @@ namespace SandboxEditor
                     }
                     eventType = MouseEvent::Up;
                 }
+                if (SandboxEditor::ManipulatorMouseWrap() && event.m_priority == ManipulatorPriority)
+                {
+                    AzToolsFramework::ViewportInteraction::ViewportMouseCursorRequestBus::Event(
+                        GetViewportId(), &AzToolsFramework::ViewportInteraction::ViewportMouseCursorRequestBus::Events::SetCursorMode,
+                        AzToolsFramework::CursorInputMode::CursorModeNone);
+                }
```

Comment:
question: Can you also do a bit of testing by moving the manipulator and then alt-tabbing to change focus and check this is called correctly? We want to ensure we avoid the cursor mode getting stuck in `CursorModeWrapped` for any reason. Basically try some stuff to see if you can break it (and think how we might test it too)